makefile

```diff
@@ -75,7 +75,7 @@ $(STATICLIB): beanstalk.o beanstalkcpp.o
 	ar -cq $@ $^
 
 $(SHAREDLIB): beanstalk.o beanstalkcpp.o
-	$(CXX) $(LINKER) -o $(SHAREDLIB)  beanstalk.o beanstalkcpp.o
+	$(CXX) $(CFLAGS) $(LINKER) $(LDFLAGS) -o $(SHAREDLIB)  beanstalk.o beanstalkcpp.o
```

Comment:
Sorry, I'm not sure what you mean. `$(LDFLAGS)` definitely needs to be passed in to the linking step when producing executables or shared libs. This is where distros put options like `-Wl,-z,-relro` and the like.

I left `LDFLAGS` undefined in the Makefile because there are currently no such flags defined. The `-L. -lbeanstack` options are okay for linking test programs and executable but they aren't real linker flags to be passed when linking the library itself hence do not belong in LDFLAGS.

Reply:
nevermind that brainfart, thanks!  :)